content/en/docs/features/monitoring-and-diagnostics/disaster-recovery.md

```diff
@@ -51,3 +51,53 @@ Having JetStream as an EventBus gives you the possibility to read stored events
 {{< warning >}}
 plgd hub doesn't guarantee delivery of all events to the EventBus. It guarantees that all events are stored in the EventStore in the correct order. In case there is a JetStream / NATS failure and plgd hub was not able to publish some events, they won't be published again and your service has to anyway fallback to reconciliation using plgd gRPC Gateway.
 {{< /warning >}}
+
+## Data Management and Failover Strategies
+
+The plgd hub is a stateful application, indicating that the data is stored in the EventStore, serving as the source of truth, which is a MongoDB. In this section, we will describe how to back up and restore data in a scenario where two clusters are running in different locations (e.g., East US / West US). The first cluster is utilized for normal operations, while the secondary cluster serves as a backup for disaster recovery.
+
+### Backup Databases
+
+To back up the database, two approaches can be used:
+
+* **Passive Backup**
+  
+  ![passive-backup](/docs/features/monitoring-and-diagnostics/static/disaster-recovery-passive-backup.drawio.svg)
+
+  The database is regularly backed up to a different location and can be used in case of failure. Although this approach is simple and requires fewer resources, the data may become outdated, and the restoration process may take some time. For MongoDB, utilize the `mongodump` tool to create a export of the database contents, store it securely, and use it in case of failure. Regular backups are essential to keep the data up-to-date. For more details on this approach, refer to the [MongoDB documentation](https://www.mongodb.com/docs/database-tools/mongodump/).
+
+* **Active Backup**
+
+  ![active-backup](/docs/features/monitoring-and-diagnostics/static/disaster-recovery-active-backup.drawio.svg)
```

Comment:
There is string "Text" next to the title "Primary cluster" (above "Sync tool"), that looks like a print demon artefact that should be removed.